snapcraft/plugins/python.py

```diff
@@ -159,9 +159,9 @@ def plugin_stage_packages(self):
             return
 
         stage_packages = [python_base]
-        if release_codename == 'bionic':
-            # In bionic, pip started requiring python-distutils to be
-            # installed.
+        # In bionic, python3's pip started requiring python-distutils
+        # to be installed.
+        if python_base == 'python3' and release_codename == 'bionic':
```

Comment:
I dunno, I'm kinda worried they'll start doing this for python2 now, too. I see no downside to leaving this the way it was, do you?

Reply:
Which is why I made my comment about checking rmadison :-)
`python-distutils` does not exist.